lightautoml/text/nn_model.py

```diff
@@ -270,9 +114,9 @@ def __init__(
         task: Task,
         torch_model: nn.Module,
         n_out: int = 1,
-        cont_embedder: Optional[Any] = None,
+        cont_embedder_: Optional[Any] = None,
```

Comment:
What is the purpose for additional post underscore?

Reply:
**params already has cont_embedder string field, so we need 1 more cont_embedder variable